setup.py

```diff
@@ -136,7 +136,7 @@ def build_extension(self, ext):
         "arrow",
         "cffi",
         "numpy==1.21.4",
-        "pandas",
+        "pandas<=1.35", # Last version that has Python 3.7 wheel
```

Comment:
my bad, it should be `<= 1.3.5`, since `1.4` satisfies `<=1.35`:

```
Searching for pandas<=1.35
Reading https://pypi.org/simple/pandas/
Downloading https://files.pythonhosted.org/packages/29/e3/6bd596d81eaf9f5b35398fdac0c535efadd9bbf8d0f859739badf9f90c63/pandas-1.4.0rc0.tar.gz#sha256=c0d453fda0a87d51f5fe65c16a89b64f13a736f4f17c0202cfcff67e6b341a57
Best match: pandas 1.4.0rc0
Processing pandas-1.4.0rc0.tar.gz
```

You can test locally with `python setup.py develop` in a new Conda environment to see if it works as expected.